prod-eu-west/services/search/main.tf

```diff
@@ -266,3 +266,45 @@ resource "aws_route53_record" "split-search" {
    ttl = "${var.ttl}"
    records = ["${data.aws_lb.default.dns_name}"]
 }
+
+// Old solr search interfaces
+
+resource "aws_lb_listener_rule" "solr-api" {
+  listener_arn = "${data.aws_lb_listener.default.arn}"
+  priority     = 80
+
+  action {
+    type = "redirect"
+
+    redirect {
+      path        = "/"
+      protocol    = "HTTPS"
+      status_code = "HTTP_301"
+    }
+  }
+
+  condition {
+    field  = "path-pattern"
+    values = ["/api*"]
```

Comment:
I would include a host condition, in case we ever use `/api` with other host names.